lib/dnsimple/client.rb

```diff
@@ -173,8 +173,8 @@ def base_options
       options = {
           format:   :json,
           headers:  {
-            'Accept' => 'application/json',
-            'User-Agent' => user_agent
+              'Accept' => 'application/json',
```

Comment:
Yes, this is similar to line continuation, although it's configurable separately.

If you prefer to revert to 2, I'm fine with that.